components/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import Slider from 'react-slick';
+import Proptypes from 'prop-types';
+import SlickSliderWrapper from './slick';
+import { colors, spacing, baseFontSize, breakpoints } from '../shared/theme';
+import Arrow from './sub-components/Arrow';
+import CarouselCard from './sub-components/CarouselCard';
+
+const MOBILE_BREAKPOINT = breakpoints.medium.width;
+
+const Carousel = props => {
+  const {
+    theme,
+    dotsPagination,
```

Comment:
on mobile dots pagination line break:

![Screen Shot 2021-03-12 at 09 43 00](https://user-images.githubusercontent.com/190265/110942099-146be080-8318-11eb-9d68-1f488c29c27c.png)

is it right? I don't know that is ok to design team.

Reply:
This problem occurs by storybook because of its padding. In real applications its not occurs.
Anyway, i did protect it hiding dots in layouts minors then 320px.
(In our layout grid, the minor layout is 460px)